src/main/webapp/app/exam/manage/exams/exam-checklist-component/exam-checklist.component.html

```diff
@@ -283,47 +283,31 @@ <h3>{{ 'artemisApp.examStatus.conduction.' + (isTestExam ? 'testExam.' : '') + '
                         <hr />
                     </div>
                     <!-- Display started exams-->
-                    <div
-                        *ngIf="
-                            examChecklist &&
-                            examChecklist.numberOfGeneratedStudentExams !== null &&
-                            examChecklist.numberOfGeneratedStudentExams !== 0 &&
-                            examChecklist.numberOfExamsStarted > 0
-                        "
-                    >
+                    <div *ngIf="examChecklist && examChecklist.numberOfGeneratedStudentExams !== null && examChecklist.numberOfGeneratedStudentExams !== 0 && numberOfStarted > 0">
                         <!-- -->
                         <span>{{ 'artemisApp.examManagement.checklist.textitems.startedExam' | artemisTranslate }}:</span>
                         <div class="pt-2">
                             <jhi-progress-bar
                                 [denominator]="examChecklist.numberOfGeneratedStudentExams!"
-                                [numerator]="examChecklist.numberOfExamsStarted!"
-                                [percentage]="(100 * examChecklist.numberOfExamsStarted!) / examChecklist.numberOfGeneratedStudentExams!"
+                                [numerator]="numberOfStarted!"
+                                [percentage]="(100 * numberOfStarted!) / examChecklist.numberOfGeneratedStudentExams!"
```

Comment:
`numberOfStarted` can not be undefined, because you already test for `> 0` above.
Please review the other occurrences as well.
```suggestion
                                [percentage]="(100 * numberOfStarted) / examChecklist.numberOfGeneratedStudentExams!"
```

Reply:
Thanks!